### DISCUSSION TITLE
Question about DC model

### DISCUSSION THREAD

**Discussion by Albizzian** [CURENT/andes#546] [Q&A] [answered]
Hello, I would like to study the dynamic characteristics of AC/DC power systems using ANDES.  I want to add a DC line between bus60 and bus61 and have added RLCs, Node, and VSCShunt models to the IEEE 68-bus system, but it does not seem to run the power flow and time-domain simulation. I checked the power flow file and found that the voltages of the Nodes are all nans. Could you help me identify the issue?  Thank you very much!
And here are the codes and attached files.·
```
ss.add("Node")
ss.add("Node")
ss.add("VSCShunt",dict(bus=60,node1='Node_1',node2="Node_2",control=0))
ss.add("VSCShunt",dict(bus=61,node1='Node_1',node2="Node_2",control=0))
ss.add("RLCs",dict(node1='Node_1',node2="Node_2"))
```

```
ss=andes.load('ieee68_DC.xlsx',setup=False)
ss.add("Alter",dict(model='PQ',dev='PQ_15',src='Ppf', t=1.0,method='*',amount=1.2))
ss.PQ.config.p2p = 1.0
ss.PQ.config.p2i = 0
ss.PQ.config.p2z = 0
ss.PQ.config.q2q = 1.0
ss.PQ.config.q2i = 0
ss.PQ.config.q2z = 0 
ss.PQ.config.pq2z = 0
ss.setup()
ss.PFlow.run()
```
![image](https://github.com/CURENT/andes/assets/78994912/87ab2e1e-a5ac-4688-a182-7cf6664c2767)
![image](https://github.com/CURENT/andes/assets/78994912/72ba083a-436e-496d-9318-a7eff0a5c203)
[ieee68_VSG_REN_DC.xlsx](https://github.com/user-attachments/files/15884008/ieee68_VSG_REN_DC.xlsx)

**Comment by jinningwang** [comment]
I suspect it is caused by unexpected usage of these models, i.e., how they are instantiated, but I didn't find a relevant case here.
Let's wait for other's input.

**Reply by Albizzian** [reply, marked as answer]
Thank you! It is actually caused by improper use of these models. By referring to the kundur_VSC.xlsx file in ANDES cases, I have successfully added the DC line into the system.  : )
[kundur_vsc.xlsx](https://github.com/user-attachments/files/15894206/kundur_vsc.xlsx)
